client/src/components/Kanban/CardForm/InnerForm/InnerForm.tsx

```diff
@@ -56,12 +58,12 @@ export const InnerForm = ({ columnId, formAction }: InnerFormProps): JSX.Element
           const isSuccess = addCard({
             name,
             columnId: columnId,
-            id: `card-${Math.floor(Math.random() * 999999)}`,
+            _id: `card-${Math.floor(Math.random() * 999999)}`,
```

Comment:
I don't love creating the id here and not just relying on mongoose, but I think that's a refactoring change we can do separately

Reply:
I see the bug. It's very peculiar as our data sets for adding cards and default cards don't match. The top one is the default and bottom is a test I did to add a card; I'll fix this issue in a new PR.

<img width="353" alt="Screen Shot 2021-06-01 at 10 02 32 AM" src="https://user-images.githubusercontent.com/35545740/120362965-7e18dc80-c2c0-11eb-82ea-032d3392d3b1.png">

Reply:
I believe adding cards was completed in @street-jackal separate PR but I'll make a fix for it merged into this one

Reply:
This is just leftover from @moffatethan's very first PR for the Board UI. We didn't have any backend set up yet to handle id creation so this was always meant to be replaced once that was in place. I used the same random id code in a bunch of my work, so that'll need to be updated too.

Reply:
Gotcha. I'll fix it up with a new PR